svo (actions dictionary).xml

```diff
@@ -1089,7 +1089,7 @@ if not next(svo.dict) then
         end,
   
         noeffect = function()
-  			  svo.rmaff('horror')
+  			  --svo.rmaff('horror')
```

Comment:
better to delete it or add a comment, otherwise when someone else comes across this in the future they will not know why is it here commented out